test/unittest/box-cc0544b6afd1.lua

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env tarantool
+local os = require('os')
+
+box.cfg {
+    listen = os.getenv('LISTEN'),
+    memtx_memory = 107374182,
+    pid_file = 'tarantool.pid',
+    force_recovery = true,
+    wal_max_size = 500
+}
+
+require('console').listen(os.getenv('ADMIN'))
```

Comment:
Why do we need a console here?

Reply:
Admin connection is used by test-run itself (say, to check `box.info.status` or for commands after `test_run:cmd("switch replica")`). Even if we got our expected fail without started console I would leave it as is. It is more or less standard instance file for test-run and it is harder to think about test-run's behaviour when it is non-standard.